tasks/main.yaml

```diff
@@ -0,0 +1,4 @@
+- include: certificate-setup.yml
```

Comment:
Doesn't this certificate-setup task install the root CA or something else? IMO, it doesn't need to be include by this role, we can use other shared role to do that.